tests/framework/microvm.py

```diff
@@ -376,10 +366,18 @@ def chroot(self):
         """Get the chroot of this microVM."""
         return self.jailer.chroot_path()
 
+    @property
+    def screen_session(self):
+        """The screen session name
+
+        The id of this microVM, which should be unique.
+        """
+        return self.id
+
     @property
     def screen_log(self):
         """Get the screen log file."""
-        return self._screen_log
+        return f"/tmp/screen-{self.screen_session}.log"
```

Comment:
right, or maybe just a `mktemp`? Unless we need to be able to access these files without the `Microvm` object laying around)